server/db.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function() {
+
+  //Requring dependencies.
+  const dotenv = require('dotenv').load();
```

Comment:
If the .env file is not present (on staging, QA, and production servers) Will this Fatally Error?